src/diffusers/models/activations.py

```diff
@@ -10,13 +10,17 @@ def get_activation(act_fn: str) -> nn.Module:
     Returns:
         nn.Module: Activation function.
     """
-    if act_fn in ["swish", "silu"]:
-        return nn.SiLU()
-    elif act_fn == "mish":
-        return nn.Mish()
-    elif act_fn == "gelu":
-        return nn.GELU()
-    elif act_fn == "relu":
-        return nn.ReLU()
+
+    activation_functions = {
+        "swish": nn.SiLU(),
+        "silu": nn.SiLU(),
+        "mish": nn.Mish(),
+        "gelu": nn.GELU(),
+        "relu": nn.ReLU(),
+    }
```

Comment:
Could we maybe define it as a constant outside of the function?

Reply:
Yes, that is a great idea.

Reply:
OK, Now update my code.

Reply:
We need to resolve the merge conflicts first.